gooddata-java/src/main/java/com/gooddata/sdk/service/project/ProjectService.java

```diff
@@ -53,6 +54,10 @@
  */
 public class ProjectService extends AbstractService {
 
+    public static final UriTemplate TEMPLATE = new UriTemplate(Project.URI);
```

Comment:
just rename to `PROJECT_TEMPLATE `

Reply:
the same question as for the other templates...